blog/_posts/2016-9-19-our-2nd-blog-post.md

```diff
@@ -0,0 +1,24 @@
+---
+title: The Miracle
+layout: post
+created_at: 2016-09-19
+permalink: blog/2016-09-19-The-Miracle
+current: blog
+author: Nada Ashraf and Mayar Alaa
+twitter: RubysSecretteam
+---
+The Miracle <3
+
+![Ruby's Secret Team](/img/blog/2016/2016-09-19-RGSoC-dayoff.jpg.jpg){RGSoC-dayoff}
```

Comment:
please add image credits to your post, as explained [here](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CONTRIBUTING.md#images).